README.md

```diff
@@ -35,6 +37,7 @@ steps:
       GH_BSN_REPO_AUTH_TOKEN: ${{GH_BSN_REPO_AUTH_TOKEN}}
       CF_BUILD_STATUS: 'success'
       GH_BSN_BUILD_CONTEXT: 'codefresh-build/ci'
+      CF_BUILD_URL: 'http://myurl.com/build/{{CF_PULL_REQUEST_NUMBER}}'
```

Comment:
Our use case is that we're creating a sandbox environment for your PR as part of this build, I want the link to take you to your sandbox environment. 
Otherwise I have the normal codefresh build status, I wouldn't need another one...

Reply:
Yeah, sure. We can include this!